packages/k8s/src/k8s/utils.ts

```diff
@@ -111,11 +111,11 @@ export function writeEntryPointScript(
   if (environmentVariables && Object.entries(environmentVariables).length) {
     const envBuffer: string[] = []
     for (const [key, value] of Object.entries(environmentVariables)) {
+      if (key.includes('=') || key.includes(`'`) || key.includes('"')) {
```

Comment:
Small Nit: Might be better to make all of them template strings for readability sake
```suggestion
      if (key.includes(`=`) || key.includes(`'`) || key.includes(`"`)) {
```

Reply:
I agree, thanks :smile: